frescobaldi_app/icons/__init__.py

```diff
@@ -42,9 +42,10 @@ def get(name):
         return _cache[name]
     except KeyError:
         icon = _cache[name] = QIcon()
-        # first try SVG
-        fname = 'icons:{0}.svg'.format(name)
-        if QFile(fname).exists():
+        # first try SVG and ico
+        if QFile(fname := 'icons:{0}.svg'.format(name)).exists():
```

Comment:
suggestion: (Saving one line and avoiding some repetitious code)
```
for ext in ['svg', 'ico']:
    if QFile(fname := f'icons:{name}.{ext}').exists():
        icon.addFile(fname)
```

Reply:
Sorry, maybe we want a break if the svg extension works - and then the line count will be the same!? :slightly_smiling_face: